tools/sctool/src/main/java/com/horizen/ScBootstrappingToolCommandProcessor.java

```diff
@@ -374,17 +373,27 @@ private void processGenerateCertWithKeyRotationProofInfo(JsonNode json) throws E
                 return;
             }
 
-            signersPublicKeys.add(pkNode.asText());
+            String pk = pkNode.asText();
+
+            int duplicateIndex = signersPublicKeys.indexOf(pk);
+            if (duplicateIndex != -1) {
+                printGenerateCertWithKeyRotationProofInfoUsageMsg(String.format("duplicated keys in signers keys are found. SignersKey with index %d equals to signersKey with index %d", duplicateIndex + 1, index));
```

Comment:
Consider changing error message to

"signersKeys contains duplicate values. SignersKey with index %d is identical to signersKey with index %d"

Reply:
Done.